cg/apps/demultiplex/sample_sheet/index.py

```diff
@@ -59,18 +61,24 @@ def get_reagent_kit_version(reagent_kit_version: str) -> str:
     return REAGENT_KIT_PARAMETER_TO_VERSION[reagent_kit_version]
 
 
-def is_reverse_complement(control_software_version: str, reagent_kit_version_string: str) -> bool:
-    """If the run used the new NovaSeq control software version (NEW_CONTROL_SOFTWARE_VERSION)
-    and the new reagent kit version (NEW_REAGENT_KIT_VERSION) the second index should be the
-    reverse complement
+def is_reverse_complement(run_parameters: RunParameters) -> bool:
+    """Return True if the second index requires reverse complement.
+
+    If the run used the new NovaSeq control software version (NEW_CONTROL_SOFTWARE_VERSION)
+    and the new reagent kit version (NEW_REAGENT_KIT_VERSION), then it requires reverse complement.
+    If the run is NovaSeqX, does not require reverse complement.
     """
+    if run_parameters.sequencer == Sequencers.NOVASEQX:
+        return False
+    control_software_version: str = run_parameters.control_software_version
+    reagent_kit_version: str = run_parameters.reagent_kit_version
     LOG.info("Check if run is reverse complement")
     if version.parse(control_software_version) < version.parse(NEW_CONTROL_SOFTWARE_VERSION):
         LOG.warning(
             f"Old software version {control_software_version}, no need for reverse complement"
         )
         return False
-    reagent_kit_version: str = get_reagent_kit_version(reagent_kit_version_string)
+    reagent_kit_version: str = get_reagent_kit_version(reagent_kit_version)
```

Comment:
I think we have settled on always using named args, so just to be consistent we might as well use it here 👍